src/components/tabs/tabs.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+    <div class="ui-tabs">
+        <ul class="ui-tabs--list">
+            <li class="ui-tabs--list-item" v-for="(tab, index) in tabs" :key="index">
+                <a :class="tabClasses(tab)"
+                        :href="tab.href"
+                        class="ui-tabs--link"
+                        tabindex="0"
+                        ref="tabRefs"
+                        @click.prevent="(e) => activateTab(e, tab)"
+                        @keypress.enter="(e) => activateTab(e, tab)"
+                        @transitionend="onTransitionEnd"
+                        @transitionstart="onTransitionStart">
+                    <render-label :label="tab.$slots['tab-label']" v-if="tab.$slots['tab-label']"/>
+                </a>
+            </li>
+        </ul>
+        <div class="ui-tabs--body" ref="content">
+            <slot/>
+        </div>
+    </div>
+</template>
+
+<script>
+    import { bemMixin } from '../../mixins';
+    
+    export default {
+        components: {
+            //If we want to render not just strings we need to render The label as a Vue Component
+            RenderLabel: {
+                functional: true,
+                render: (h, ctx) => ctx.props.label,
+            },
+        },
+        mixins: [
+            bemMixin('ui-tabs'),
+        ],
+        props: {
+            selectedTab: {
```

Comment:
I'd go with `selectedTabId` here, because that's what this value is actually holding.

Reply:
changed as requested